interoperator/internal/renderer/helm/samples/postgresql/crds/sample.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: abc
```

Comment:
What's this?

Reply:
Some sample for testing used in the unit tests